Tests/SentryProfilerTests/SentryContinuousProfilerTests.swift

```diff
@@ -0,0 +1,53 @@
+import _SentryPrivate
+@testable import Sentry
+import SentryTestUtils
+import XCTest
+
+#if os(iOS) || os(macOS) || targetEnvironment(macCatalyst)
+
+final class SentryContinuousProfilerTests: XCTestCase {
+    private var fixture: SentryProfileTestFixture!
+    
+    override class func setUp() {
+        super.setUp()
+        SentryLog.configure(true, diagnosticLevel: .debug)
+    }
+    
+    override func setUp() {
+        super.setUp()
+        fixture = SentryProfileTestFixture()
+    }
+    
+    func testStartingAndStoppingContinuousProfiler() throws {
+        try performContinuousProfilingTest()
+    }
+}
+
+private extension SentryContinuousProfilerTests {
+    func performContinuousProfilingTest() throws {
+        SentryContinuousProfiler.start()
+        
+        XCTAssert(SentryContinuousProfiler.isCurrentlyProfiling())
+        
+        try addMockSamples()
+        SentryContinuousProfiler.stop()
+        
+        XCTAssertFalse(SentryContinuousProfiler.isCurrentlyProfiling())
```

Comment:
`m`: Any reason you can't use Nimble assertions here?

Reply:
Habit

Reply:
Okay, then, let's revisit this. I like them a lot. The most important thing is that we all on the Cocoa SDK agree on using Nimble or not. Let's discuss this in Slack.